File: Latihan2/Python/Product.py
#kelas yang digunakan untuk mengimplementasikan Product
class Product:
    def __init__(self):
        #constructor
        self.price = 0
        self.idProduct = 0

    def setPrice(self, price):
        #mengeset nilai atribut price
        self.price = price

    def getPrice(self):
        #return nilai atribut price
        return self.price

    def setIdProduct(self, idProduct):
        #mengeset nilai atribut idProduct
        self.idProduct = idProduct

    def getIdProduct(self):
        #return nilai atribut idProduct
        return self.idProduct

    def printProduct(self):
        #menampilkan atribut dari Product
        print("Id Product: " + str(self.getIdProduct()))
        print("Price: " + str(self.getPrice()))

File: Latihan2/C++/Product.cpp
#include <string>
#include <iostream>
using namespace std;

//kelas yang digunakan untuk mengimplementasikan Product
class Product {
private:
    //private attribute dari kelas Product
    int price;
    int idProduct;

public:
    //constructor
    Product() {}

    //mengeset nilai atribut price
    void setPrice(int price) {
        this->price = price;
    }

    //return nilai atribut price
    int getPrice() {
        return this->price;
    }

    //mengeset nilai atribut idProduct
    void setIdProduct(int idProduct) {
        this->idProduct = idProduct;
    }

    //return nilai atribut idProduct
    int getIdProduct() {
        return this->idProduct;
    }

    //menampilkan atribut dari Product
    void printProduct() {
        cout << "Id Product : " << this->getIdProduct() << endl;
        cout << "Price : " << this->getPrice() << endl;
    }

    //destructor
    ~Product() {}
};